infra/config.go

```diff
@@ -22,14 +22,20 @@ type PgConfig struct {
 	User                string
 	MaxPoolConnections  int
 	ClientDbConfigFile  string
+	SslMode             string
 }
 
 func (config PgConfig) GetConnectionString() string {
 	if config.ConnectionString != "" {
 		return config.ConnectionString
 	}
-	connectionString := fmt.Sprintf("host=%s user=%s password=%s database=%s sslmode=disable",
```

Comment:
"sslmode=disable" was not great. Replace it by "prefer" at least below (by default)